AVInput/AVInput.cpp

```diff
@@ -47,6 +47,7 @@
 #define AVINPUT_METHOD_GET_EDID_VERSION "getEdidVersion"
 #define AVINPUT_METHOD_SET_EDID_ALLM_SUPPORT "setEdid2AllmSupport"
 #define AVINPUT_METHOD_GET_EDID_ALLM_SUPPORT "getEdid2AllmSupport"
+#define AVINPUT_METHOD_SET_AUDIO_MIXER_LEVELS "setAudioMixerLevels"
```

Comment:
Please check the architecture for API name correction:  https://www.stb.bskyb.com/confluence/pages/viewpage.action?spaceKey=2016&title=HDMI+Audio+Mixing